# a msg type for position data from analyze_phase.py
Header header
float32 x_position
float32 y_position
float32 z_position
float32 yaw_angle
